docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,115 @@
+# How to Understand Strategies Descriptions
+
+Navigating strategies descriptions at [yearn.watch↗](https://yearn.watch/) or [vaults.yearn.finance↗](https://vaults.yearn.finance/) may expose you to terms that you have never seen, this page contains a complete glossary on what each term means when used to describe strategies for yearn vaults.
```

Comment:
While reading the strategy descriptions at [yearn.watch↗](https://yearn.watch/) or [vaults.yearn.finance↗](https://vaults.yearn.finance/) you may come across unfamiliar (and if we're honest, often confusing) terms. But do not fear, the Yearn glossary is here to help; providing a comprehensive travel guide to the brave new world of DeFi.